### ISSUE TITLE
Longer tips

### ISSUE THREAD

**Issue by Beliavsky** [zmoon/FortranTipBrowser#2]
Thanks for your project.

My workflow is to 

(1) write a code foo.f90
(2) write some text in foo.txt
(3) shorten the text to get it to 257 characters (since the link to code takes 23 characters). Honestly, this is a pain :).
(4) tweet the code with text using Carbon app

Going forward, I could upload to GitHub the result of step (2) before shortening. The text would not be long, likely 1/2 page or less. A web site does not need to be bound to Twitter's strict character limits. Given the naming convention it would be clear which longer tips correspond to which codes and tweets.

I don't know if your FTB could use longer tips. If not, that's fine.

**Comment by zmoon** [comment]
Yeah, that would be great! Though I haven't done too much yet, that is one of my goals, to add more to the text, maybe provide more notes/references related to the tip.
